pkg/resource/podtemplatespec.go

```diff
@@ -111,6 +111,7 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, params *parameters.
 	}
 
 	deps := newDependencies()
+	deps.AddSecret(v1alpha1.ImageRegistryPrivateConfigurationUser)
```

Comment:
shouldn't this be `v1alpha1.ImageRegistryPrivateConfiguration` ?

Reply:
No, it's already watching that ImageRegistryPrivateConfiguration secret

Reply:
Fixed in current push